e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts

```diff
@@ -258,6 +258,28 @@ class RegistrationDetails {
     }
   }
 
```

Comment:
Your RegistrationDetailsPage.ts file is becoming quite long. Maybe it makes sense to consider creating a seperate RegistrationPhysicalCardOverview.ts file and move all card functionality to there.  

I am not 100% sure about the structure you should try to adhere to in playwright e2e code. But for example in our fronted the RegistrationDetailsPage is seperated out in a lot of components .

Reply:
That is actually a good idea to do that. Thanks

Reply:
I have added new file and according to your input I'll add more files depending on functionality.